app/src/main/java/com/github/neighbortrader/foodboardapp/MainActivity.java

```diff
@@ -13,17 +19,35 @@
 import androidx.navigation.ui.NavigationUI;
 
 import com.crashlytics.android.Crashlytics;
+import com.github.neighbortrader.foodboardapp.clientmodel.Offer;
+import com.github.neighbortrader.foodboardapp.requests.GetAllOffersRequest;
+import com.github.neighbortrader.foodboardapp.requests.OnEventListener;
 import com.google.android.material.floatingactionbutton.FloatingActionButton;
 import com.google.android.material.navigation.NavigationView;
 import com.google.android.material.snackbar.Snackbar;
 
+import java.util.List;
+
 import io.fabric.sdk.android.Fabric;
 
 public class MainActivity extends AppCompatActivity {
 
+    public static String TAG = MainActivity.class.getSimpleName();
+
     private AppBarConfiguration mAppBarConfiguration;
```

Comment:
für was steht das m bei mAppBarConfiguration ?

Reply:
m = member 
Ist auto generierter-Code... würde ich tatsächlich nicht so schrieben in Java wird aber im Android-Kontext häuig gemacht. Ich finde grundsätzlich `this.appBarCon...` deutlich angenehmer